### PR TITLE
Add June 22 meeting minutes

### PR DESCRIPTION
**Pull request by renrutnnej**
Sorry for the delay folks! I asked @nebrius for some formatting feedback before PRing.

### CODE REVIEW

File: meetings/2017-06-22.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+Tracy Hinds: I’m not wanting to delay process for these two at all. We’ve discussed a simple path: observe meetings for 3 months, meet the minimum participation goal, and then they’ll get to join as member. How it worked with our first round of members was whoever volunteered, basically got added. After that we discussed a process, but never finalized it, so let’s PR a process, and add this second round as members.  
+Michael Dawson: In other working groups, whoever showed up gets joined (as a member) and once the groups were more formalized, a membership process was established.  
+Tracy: Let’s not impede getting the second group added.  
+Bryan: Let’s add the second round but also get our membership process documented for the future. What do the volunteers think?  Jenn Turner: Don’t have a preference either way. Let’s stick to whatever process we decide.  
+Bryan: Let’s add this second round of membership including Jenn, Tierney, Olivia Hugger, and potentially Richard Littauer, if he’s interested, and then try and get a basic process in place for the next round.
```
Comment (RichardLitt) [inline meetings/2017-06-22.md]: I'm interested? I think I said thanks here.

**Comment by renrutnnej** [comment]
Thanks @RichardLitt ☺️

File: meetings/2017-06-22.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
+# Node.js Foundation Community Committee Meeting 2017-06-22
+ 
+## Links
+ 
+* **GitHub Issue**: [community-committee#80](https://github.com/nodejs/community-committee/issues/80)
+* **Minutes Google Doc**: https://docs.google.com/document/d/14Pd6e40n1wPEuRBknDJUFwlf9XDhWI94jCgMEvTfVS4/
+* _Previous Minutes Google Doc_:
+https://docs.google.com/document/d/1J14T5lXuFWsaTqfkpmPruAZ4E-WsVe2V5TAaKwDjHN0/
+* _Video Recording_: https://www.youtube.com/watch?v=gwSWLXD2nLw
```
Comment (bnb) [inline meetings/2017-06-22.md]: Total minor nit, but should the video recording formatting (italic) be changed to match the formatting of GitHub Issue and Minutes Doc (boldface)?

Comment (renrutnnej) [inline meetings/2017-06-22.md]: I have no preference either way, I left the formatting the way I found it ¯\_(ツ)_/¯

Comment (nebrius) [inline meetings/2017-06-22.md]: Yeah, I just kinda randomly picked one when creating it. There was pretty close to zero thought put into it ;-)

**Pull request merged**